web/src/components/moreButton.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function MoreButton({ text, isDisabled, isLeft }) {
+    const disabledStyle = 'bg-brand-9';
```

Comment:
I don't think these are the correct styles for a deactivated button on a dark background. Please check and confirm

Reply:
Check the active styles as well please